example/take_movie.py

```diff
@@ -48,6 +50,18 @@
                     help='Number of episodes of expert trajectories.')
 parser.add_argument('--ddpg', action='store_true',
                     default=False, help='If True, policy for DDPG is used.')
+parser.add_argument('--num_iter', type=int, default=2,
+                    help='Number of iteration of CEM.')
+parser.add_argument('--num_sampling', type=int, default=60,
+                    help='Number of samples sampled from Gaussian in CEM.')
+parser.add_argument('--num_best_sampling', type=int, default=6,
+                    help='Number of best samples used for fitting Gaussian in CEM.')
+parser.add_argument('--eps', type=float, default=0.2,
+                    help='Probability of random action in epsilon-greedy policy.')
+parser.add_argument('--save_memory', action='store_true', default=False,
+                    help='If true, save memory while need more computation time by for-sentence.')
+parser.add_argument('--cem', action='store_true',
+                    default=False, help='If True, policy for cross entropy method is used.')
```

Comment:
@TomeHirata 
The name of this option `cem` is not consistent with `ddpg`.
Could you change the name into `qtopt`？